client/lib/media/library-selected-store.js

```diff
@@ -13,24 +13,25 @@ var MediaStore = require( './store' ),
 /**
  * Module variables
  */
-var _media = {},
```

Comment:
Yes, good idea. Though it's tricky as by default we export object and JS modules are transpiled to node modules, but I can give it a try 👍